pulpcore.fc

```diff
@@ -4,6 +4,8 @@
 /usr/local/lib/pulp/bin/gunicorn		--	gen_context(system_u:object_r:pulpcore_exec_t,s0)
 /usr/local/lib/pulp/bin/rq		--	gen_context(system_u:object_r:pulpcore_exec_t,s0)
 
+/usr/libexec/pulpcore(/.*)?	--	gen_context(system_u:object_r:pulpcore_exec_t,s0)
```

Comment:
this is already present:
https://github.com/pulp/pulpcore-selinux/blob/32edc8fb42ef18c1257bac473bfde11a45bf28e7/pulpcore.fc#L4